docs/reference/pfazure-command-reference.md

```diff
@@ -303,6 +304,10 @@ pfazure run stream --name
 
 Name of the run.
 
+`--timeout`
+
+Timeout in seconds. If the run keeps in status `NotStarted` longer than the timeout value, the stream operation will abort. Default value is 300 seconds
```

Comment:
We should not check NotStarted if we intro this generic argument

Reply:
Change to this:

`If the run stays in the same status and produce no new logs in a period longer than the timeout value, the stream operation will abort. Default value is 300 seconds`